Applications/DataExplorer/VtkVis/NetCdfConfigureDialog.cpp

```diff
@@ -94,9 +94,9 @@ void NetCdfConfigureDialog::on_comboBoxDim1_currentIndexChanged(int id)
 //set up y-axis/lon
 void NetCdfConfigureDialog::on_comboBoxDim2_currentIndexChanged(int id)
 {
+    Q_UNUSED(id);
     if (_currentVar.getDimCount() > 1)
     {
-        if (id == -1) id = 0;
```

Comment:
style: could be directly commented in the arguments list.
issue: same unneeded assignment above in `on_comboBoxDim1_currentIndexChanged`.